From 9256f87498d1471983a80a3fdfa8035205708d15 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 18 Jun 2020 08:22:25 -0500
Subject: [PATCH 6/6] configs: beacon-rzg2m.h: Fix loading addresses

The default settings for loadaddr work with newer kernels, but fail
to boot the Beacon master kernel (4.19) due to the location the
kernel is being loaded.

This patch moves the fdt_addr and loadaddr back to their defaults
of 0x48080000 and 0x48000000 resepctively which restores booting
from the older kernel.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 include/configs/beacon-rzg2m.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/beacon-rzg2m.h b/include/configs/beacon-rzg2m.h
index e788a4ee82..3d51efa195 100644
--- a/include/configs/beacon-rzg2m.h
+++ b/include/configs/beacon-rzg2m.h
@@ -29,7 +29,8 @@
 	"script=boot.scr\0" \
 	"image=Image\0" \
 	"console=ttySC0,115200\0" \
-	"fdt_addr=0x48080000\0"			\
+	"fdt_addr=0x48000000\0"			\
+	"loadaddr=0x48080000\0"	\
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fdt=try\0" \
 	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0" \
-- 
2.17.1

